data_structures/binary_tree/binary_search_tree.py

```diff
@@ -107,6 +107,44 @@ def get_min(self, node=None):
                 node = node.left
         return node
 
+    def get_height(self, node) -> int:
+        r"""
+        Param: node
+        --------
+        Returns: height
+        --------
+        Height of a binary search tree is the number of nodes present in
+        the longest path from the root node down to the farthest leaf node.
+        --------
+        Example
+
+                  9
+                 / \
+                3   10
+               / \    \
+              2   6    14
+                 / \
+                4   7
+
+        >>> bst = BinarySearchTree().insert(9, 3, 6, 2, 10, 14, 4, 7)
+        >>> bst.get_height(bst.root)
+        4
+        """
+        if node is None:
+            return 0
+
+        else:
+
+            # get depth of each subtree, recursive apporach
+            left_height = self.get_height(node.left)
+            right_height = self.get_height(node.right)
+
+            # returns the largest height among both subtree
+            if left_height > right_height:
+                return left_height + 1
+            else:
+                return right_height + 1
+
     def remove(self, value):
```

Comment:
Include typehints

Reply:
Could you please help me out, where should I include typehints.
Did you mean about the method remove(). If so, I can include typehint for that too.

Reply:
I do not know why there were some lines highligthed, I did not want to do that.

I am not a maintainer in this repository. But I guess they will ask you to include type hints in all the file.

Reply:
Thanks for the suggestion, I've included all possible typehints as far as I know.